### PR TITLE
Loosen manageiq-password to < 2

### PR DESCRIPTION
**Pull request by Fryguy**
Part of https://github.com/ManageIQ/manageiq/pull/21178

### CODE REVIEW

File: manageiq-appliance_console.gemspec [hunk 1]
```diff
@@ -28,7 +28,7 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency "highline",                "~> 1.6.21"
   spec.add_runtime_dependency "i18n",                    "~> 0.8"
   spec.add_runtime_dependency "linux_admin",             "~> 2.0"
-  spec.add_runtime_dependency "manageiq-password",       "~> 0.3"
+  spec.add_runtime_dependency "manageiq-password",       "~> 1.0"
```
Comment (bdunne) [inline manageiq-appliance_console.gemspec:31]: Can this be loosened to `< 2`?

**Comment by Fryguy** [comment]
@bdunne Updated

**Comment by kbrock** [comment]
looks like a bad test. it is expecting a v1 key

**Comment by Fryguy** [comment]
@bdunne Updated

**Pull request merged**